From 3b79790375b233c44fc0531617ca4230de590f84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mos=C3=A8=20Giordano?= <mose@gnu.org>
Date: Wed, 22 Jun 2022 10:44:40 +0100
Subject: [PATCH 1/1] Fix signature of `SBase_getNumPlugins` in prototype

This was different from signature in implementation, leading to the
implementation not being C-exported correctly, and having C++ name mangling:

```console
% nm libsbml.so | grep SBase_getNumPlugins
00000000008f5240 T _Z19SBase_getNumPluginsPK5SBase
``
---
 src/sbml/SBase.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/sbml/SBase.h b/src/sbml/SBase.h
index 11bc1a1d8..ce7e00f08 100644
--- a/src/sbml/SBase.h
+++ b/src/sbml/SBase.h
@@ -5074,7 +5074,7 @@ SBase_hasValidLevelVersionNamespaceCombination(SBase_t *sb);
  */
 LIBSBML_EXTERN
 int
-SBase_getNumPlugins(SBase_t *sb);
+SBase_getNumPlugins(const SBase_t *sb);
 
 /**
  * Returns a plug-in structure (extension interface) for an SBML Level&nbsp;3
-- 
2.36.1

